test/Unit/Durable/DurableTimerTests.cs

```diff
@@ -143,6 +204,13 @@ private void EmulateStop(DurableTimer durableTimer)
             durableTimer.Stop();
         }
 
+        private void VerifyCreateDurableTimerActionAdded(OrchestrationContext context, DateTime fireAt)
+        {
+            var actions = DurableTestUtilities.GetCollectedActions(context);
+            var action = (CreateDurableTimerAction)actions[actions.Count - 1];
```

Comment:
Use `.Last()` instead of `[Count - 1]`: this is simpler and communicates the intent better.